frontend/packages/knative-plugin/src/topology/components/knativeComponentUtils.ts

```diff
@@ -74,11 +77,23 @@ export const isEventPubSubDroppable = (source: Node, target: Node) => {
   );
 };
 const getKnativeTooltip = (monitor): string => {
-  return monitor.getOperation()?.type === CREATE_PUB_SUB_CONNECTOR_OPERATION
-    ? monitor.getItem()?.getData()?.resources.obj.kind === EventingBrokerModel.kind
-      ? i18next.t('knative-plugin~Add Trigger')
-      : i18next.t('knative-plugin~Add Subscription')
-    : i18next.t('knative-plugin~Move sink to service');
+  switch (monitor.getOperation()?.type) {
+    case CREATE_PUB_SUB_CONNECTOR_OPERATION: {
+      return monitor.getItem()?.getData()?.resources.obj.kind === EventingBrokerModel.kind
+        ? i18next.t('knative-plugin~Add Trigger')
+        : i18next.t('knative-plugin~Add Subscription');
+    }
+    case MOVE_PUB_SUB_CONNECTOR_OPERATION: {
+      return monitor.getItem()?.getData()?.resources.obj.kind === EventingBrokerModel.kind
+        ? i18next.t('knative-plugin~Move Trigger')
+        : i18next.t('knative-plugin~Move Subscription');
+    }
+    case MOVE_EV_SRC_CONNECTOR_OPERATION: {
+      return i18next.t('knative-plugin~Move sink to service');
```

Comment:
should the type be dynamic now as drop can be ksvc or kafkaSink, currently if I create a pingSource  -> kafkaSink A and try to drag the connector and drop on another kafkaSink then tooltip says `Move sink to service`

![KStooltip2](https://user-images.githubusercontent.com/5129024/186110024-adbe5a33-91fd-4e54-9d9e-e6a15928f371.gif)

Reply:
Done